devices/A4988/A4988 .cs

```diff
@@ -58,16 +59,16 @@ protected virtual void SleepBetweenSteps()
         /// <summary>
         /// Rotates a stepper motor.
         /// </summary>
-        /// <param name="degree">Rotation in degrees.</param>
-        public virtual void Rotate(double degree)
+        /// <param name="angle">Angle to rotate.</param>
+        public virtual void Rotate(UnitsNet.Angle angle)
         {
-            if (degree == 0)
+            if (angle.Value == 0)
             {
                 return;
             }
 
-            _dirPin.Write(degree > 0 ? PinValue.High : PinValue.Low);
-            var degreeForStepsCalculation = degree < 0 ? -degree : degree;
+            _dirPin.Write(angle.Value > 0 ? PinValue.High : PinValue.Low);
```

Comment:
angle.Value will check the value of the angle based on the way it's been created.
So better use ange.Degree instead.